### PR TITLE
Lovelace: Allow hiding tabs

### PR DESCRIPTION
**Pull request by iantrich**
**Description:**

https://github.com/home-assistant/home-assistant-polymer/pull/3811

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/lovelace/views.markdown [hunk 1]
```diff
@@ -50,6 +50,10 @@ views:
       required: false
       description: Themes view and cards, more info below.
       type: string
+    hide:
+      required: false
+      description: "Hide the view from all users or a list of users to hide the view from."
+      type: [boolean, list]
```
Comment (frenck) [inline source/lovelace/views.markdown:56]: Since the type "boolean" is here, it requires a `default:` item in the configuration.

**Review by klaasnicolaas — DISMISSED** [review]
LGTM

**Comment by iantrich** [comment]
parent has been merged

**Review by frenck — APPROVED** [review]
Thanks, @iantrich! 👍 

Happy Hacktoberfest 🎉

**Pull request merged**